15 Jun 2022 22:01:09,652 ERROR   Error: An error occurred while checking for chart dependencies. You may need to run `helm dependency build` to fetch missing dependencies: found in Chart.yaml, but missing in charts/ directory: alfresco-content-common
15 Jun 2022 22:01:09,653 ERROR   Error running template command. Exit Value = 1.
15 Jun 2022 22:01:09,653 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,azuregateway.enabled=false,b2gateway.enabled=false,global.service.name=foo-name,global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,gcsgateway.enabled=true,ingress.enabled=false,ingress.domain=foo-domain,insightzeppelin.enabled=true,insightKey=foo-key,insightCrt=foo-crt,jfmcCrt=foo,jfmcKeystore=foo,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true,mongodb.db.adminPassword=foo-password,mongodb.db.mcPassword=foo-password,mongodb.db.insightPassword=foo-password,mongodb.mongodbDatabase=foo,mongodb.mongodbPassword=foo-password,mongodb.mongodbRootPassword=foo-password,mongodb.mongodbUsername=foo-username,mongodb.uri=http://foo-uri,redis.redisPassword=foo-password,replicationPassword=foo-password,s3gateway.enabled=true,service.type=foo,service.port.number=9999,signalsciences.accessKeyId=foo-key,signalsciences.accessKeyIdExistingSecret=foo-secret,superuserPassword=foo-password"  template alfresco-elasticsearch-connector 
15 Jun 2022 22:01:09,654 ERROR   Exception rendering template for alfresco-elasticsearch-connector:0.1.0 : Error running template command. Exit Value = 1.
15 Jun 2022 22:01:09,654 ERROR   Error resolving chart dependencies: Exception rendering template for alfresco-elasticsearch-connector:0.1.0 : Error running template command. Exit Value = 1.
15 Jun 2022 22:02:02,980 ERROR   Chart alfresco-content-services:5.3.0-M1 printed successfully with the refresh option. See the prior errors that occured without the refresh option to understand why refresh was needed.
15 Jun 2022 22:03:54,306 ERROR   Error: An error occurred while checking for chart dependencies. You may need to run `helm dependency build` to fetch missing dependencies: found in Chart.yaml, but missing in charts/ directory: common
15 Jun 2022 22:03:54,307 ERROR   Error running template command. Exit Value = 1.
15 Jun 2022 22:03:54,307 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,azuregateway.enabled=false,b2gateway.enabled=false,global.service.name=foo-name,global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,gcsgateway.enabled=true,ingress.enabled=false,ingress.domain=foo-domain,insightzeppelin.enabled=true,insightKey=foo-key,insightCrt=foo-crt,jfmcCrt=foo,jfmcKeystore=foo,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true,mongodb.db.adminPassword=foo-password,mongodb.db.mcPassword=foo-password,mongodb.db.insightPassword=foo-password,mongodb.mongodbDatabase=foo,mongodb.mongodbPassword=foo-password,mongodb.mongodbRootPassword=foo-password,mongodb.mongodbUsername=foo-username,mongodb.uri=http://foo-uri,redis.redisPassword=foo-password,replicationPassword=foo-password,s3gateway.enabled=true,service.type=foo,service.port.number=9999,signalsciences.accessKeyId=foo-key,signalsciences.accessKeyIdExistingSecret=foo-secret,superuserPassword=foo-password"  template alfresco-sync-service 
15 Jun 2022 22:03:54,307 ERROR   Exception rendering template for alfresco-sync-service:3.0.9 : Error running template command. Exit Value = 1.
15 Jun 2022 22:03:54,307 ERROR   Error resolving chart dependencies: Exception rendering template for alfresco-sync-service:3.0.9 : Error running template command. Exit Value = 1.
15 Jun 2022 22:04:40,087 ERROR   Chart alfresco-content-services:5.2.0 printed successfully with the refresh option. See the prior errors that occured without the refresh option to understand why refresh was needed.
